.github/workflows/ci.yml

```diff
@@ -2,21 +2,45 @@ name: ci
 
 on:
   push:
-    tags:
-      - v*
+    branches:
+      - master
+  pull_request:
+
+env:
+  NODE_VERSION: '10'
 
 jobs:
-  deploy:
-    name: Deploy
+  test:
+    name: Run Tests
+    runs-on: ${{ matrix.os }}
 
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        browser: [Chrome]
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Publish
-      run: |
-        echo "//npm.pkg.github.com/:_authToken=$GITHUB_TOKEN" >> ~/.npmrc
-        npm publish --access public
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/checkout@v2
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - uses: actions/cache@v2
+      id: yarn-cache
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '${{ env.NODE_VERSION }}'
+    - uses: webfactory/ssh-agent@v0.5.0
```

Comment:
Is there a way to use a github token instead of a SSH key ? 
On GH actions, tokens are shortlived, so it could be great to use this ?